core/src/main/java/org/mskcc/cbio/oncokb/util/MainUtils.java

```diff
@@ -141,6 +142,19 @@ public static MutationEffect findHighestMutationEffect(Set<MutationEffect> mutat
         return index == 100 ? null : PRIORITIZED_MUTATION_EFFECTS.get(index);
     }
 
+    public static IndicatorQueryMutationEffect findHighestMutationEffectByEvidence(Set<Evidence> evidences) {
+        Integer index = 100;
+        IndicatorQueryMutationEffect indicatorQueryMutationEffect = new IndicatorQueryMutationEffect();
+        for (Evidence evidence : evidences) {
+            MutationEffect mutationEffect = MutationEffect.getByName(evidence.getKnownEffect());
+            if (PRIORITIZED_MUTATION_EFFECTS.indexOf(mutationEffect) < index) {
```

Comment:
Shouldn't we update the "index" value to make the comparison valid? In this case, it is alway 100

Reply:
Ah yeah yeah, thanks!